cocos/2d/framework/renderable-2d.ts

```diff
@@ -111,11 +110,11 @@ export enum InstanceMaterialType {
  * @zh 所有支持渲染的 2D 组件的基类。
  * 这个组件会设置 [[Node]] 上的 [[NodeUIProperties.uiComp]]。
  */
-@ccclass('cc.Renderable2D')
+@ccclass('cc.UIRenderer')
 @requireComponent(UITransform)
 @disallowMultiple
 @executeInEditMode
-export class Renderable2D extends RenderableComponent {
+export class UIRenderer extends BaseRenderer {
```

Comment:
Please keep using `Renderable` instead of `Renderer`, they are objects which can be rendered, so they are renderable, but they are not the renderer, they are not rendering anything, they are only providing data to the renderer